newrelic/structures_newrelic_alert_muting_rule.go

```diff
@@ -85,22 +257,28 @@ func expandMutingRuleValues(values []interface{}) []string {
 
 func flattenMutingRule(mutingRule *alerts.MutingRule, d *schema.ResourceData) error {
 
-	x, ok := d.GetOk("condition")
+	x := d.Get("condition")
```

Comment:
Was this change intentional?

Reply:
Yeah, we the previous code wasn't ever doing anything with the value of `ok` -- my ide highlighted it as unused. We were passing it into the flatten function and then ignoring it.